package-lock.json

```diff
@@ -1,3352 +1,8 @@
 {
   "name": "kodcoach.se",
   "version": "0.1.0",
-  "lockfileVersion": 2,
```

Comment:
This big diff is caused by inconsistens Node version usage. (The previous `package-lock.json` was probably created with a higher version of node than what is used in `deploy.yml` which might cause incompatibilities.)

This lock file is generated using Node 12.